plugins/redmine_bugs_ruby_lang/lib/redmine_mailing_list_integration/redmine_ext/mailer.rb

```diff
@@ -49,26 +43,29 @@ def record_message(issue, journal, mailing_lists)
   end
 end
 
-Mailer.class_eval do
-  class << self
-    def deliver_issue_add(issue)
-      unless issue.originates_from_mail?
-        issue_add(issue.author, issue).deliver_later
-      end
+module MailingListIntegrationMailerClass
+  def deliver_issue_add(issue)
+    unless issue.originates_from_mail?
+      super
+      issue_add_mailing_list(issue.author, issue).deliver_later
     end
+  end
 
-    def deliver_issue_edit(journal)
-      unless journal.originates_from_mail?
-        issue_edit(journal.issue.author, journal).deliver_later
-      end
+  def deliver_issue_edit(journal)
+    unless journal.originates_from_mail?
+      super
+      issue_edit_mailing_list(journal.issue.author, journal).deliver_later
     end
+  end
 
-    def deliver_attachments_added(attachments)
-      attachment = attachments.first
-      return unless attachment.container_type == 'Issue'
-      attachments_added(attachment.container.author, attachments).deliver_later
-    end
+  def deliver_attachments_added(attachments)
+    attachment = attachments.first
+    return unless attachment.container_type == 'Issue'
```

Comment:
I don't understand what is the reason to patch deliver_attachments_added ? AFAIK it's meant for projects and documents, it could never be an Issue.